src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -636,6 +636,9 @@ def get_provisioning_state(instance):
             properties = getattr(instance, 'properties', None)
             if properties:
                 provisioning_state = getattr(properties, 'provisioning_state', None)
+                if not provisioning_state:
+                    additional_properties = getattr(properties, 'additional_properties', {})
+                    provisioning_state = additional_properties.get('provisioningState')
```

Comment:
For this case, is `privisioningState` always existing ?

Reply:
@arrownj This is a bug in core. The `provisioningState` label for keyvault is under `properties.provisioning_state.additional_properties`, the original code was not able to read it.

Reply:
better to add some comments here to explain it